arm-compute/2017-03-30/swagger/runCommands.json

```diff
@@ -198,6 +198,13 @@
           "type": "string",
           "description": "The run command id."
         },
+        "script": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The script to be executed."
```

Comment:
script parameter is optional. All commands (associated with commandId) has a pre-defined scripts already. But this gives you the ability to specify(override) your own script especially when you run customScript command.
API can validate the parameter and throws error is the script should not be set or if it is required.